.golangci.yml

```diff
@@ -9,56 +9,46 @@ linters-settings:
       - '^log\.'
       - '^print$'
       - '^println$'
+      - '^panic$'
+  godox:
+    # TODO, OPT, etc. comments are fine to commit. Use FIXME comments for
+    # temporary hacks, and use godox to prevent committing them.
+    keywords: [FIXME]
+  importas:
+    no-unaliased: true
+  varnamelen:
+    ignore-decls:
+      - T any
+      - i int
+      - wg sync.WaitGroup
 linters:
-  enable:
-    - asciicheck
-    - bidichk
-    - bodyclose
-    - containedctx
-    - contextcheck
-    - decorder
-    # - depguard
-    - dogsled
-    - errcheck
-    - execinquery
-    - exportloopref
-    - forbidigo
-    - forcetypeassert
-    - gochecknoinits
-    - gofmt
-    - goheader
-    - goimports
-    - gomodguard
-    - goprintffuncname
-    - gosec
-    - gosimple
-    - gosmopolitan
-    - govet
-    - grouper
-    - importas
-    - inamedparam
-    - ineffassign
-    - interfacebloat
-    - logrlint
-    - makezero
-    - mirror
-    - misspell
-    - nakedret
-    - nilerr
-    - nolintlint
-    - nosprintfhostport
-    - perfsprint
-    - predeclared
-    - promlinter
-    - protogetter
-    - reassign
-    - rowserrcheck
-    - staticcheck
-    - stylecheck
-    - testifylint
-    - typecheck
-    - unconvert
-    - unused
-    - wastedassign
-    - whitespace
-  disable-all: true
+  enable-all: true
+  disable:
+    - cyclop            # covered by gocyclo
+    - deadcode          # abandoned
```

Comment:
I think we want to exclude depguard - newer versions require explicit config to depend on anything outside of the Go stdlib.

Reply:
Done